addons/binding/org.openhab.binding.innogysmarthome/.classpath

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/google-oauth-client-1.22.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/joda-time-2.9.4.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/google-http-client-jackson2-1.22.0.jar" sourcepath="/Users/ollie-dev/.m2/repository/com/google/http-client/google-http-client-jackson2/1.22.0/google-http-client-jackson2-1.22.0-sources.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/google-http-client-1.22.0.jar"/>
+	<classpathentry kind="lib" path="lib/httpclient-4.0.1.jar"/>
+	<classpathentry kind="lib" path="lib/httpcore-4.0.1.jar"/>
+	<classpathentry kind="lib" path="lib/jackson-core-2.1.3.jar"/>
+	<classpathentry kind="lib" path="lib/jsr305-1.3.9.jar"/>
```

Comment:
In order to be able to resolve the Eclipse JDT annotations you have to provide the following optional import statement to your MANIFEST.MF: `org.eclipse.jdt.annotation;resolution:=optional,`.

Reply:
Added the import. Now I should annotate every function with those annotations, right? Sounds reasonable...

Reply:
Please have a look at the coding guidelines. We very recently did document the new default for null annotations, which is @NonNullByDefault on the class level.
Since its not yet online here is the link to the PR for the documentation: https://github.com/eclipse/smarthome/pull/4275/files#diff-24ba7e763168d1631a43ebdef19bcd5dR27

Reply:
docs just went online, see https://eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style

Reply:
Since I added jdt annotation, I get the following error using `mvn clean install` though the project compiles fine in Eclipse:
```
[INFO] {osgi.os=linux, osgi.ws=gtk, org.eclipse.update.install.features=true, osgi.arch=x86}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.openhab.binding.innogysmarthome 2.2.0.qualifier
[ERROR]   Missing requirement: org.openhab.binding.innogysmarthome 2.2.0.qualifier requires 'package org.eclipse.jdt.annotation 0.0.0' but it could not be found
```
Do you have any idea, what is missing here? I didn't find a solution yet. However, I did not use any of the named annotations yet and only imported the package in MANIFEST.MF.

Reply:
Now I tried to add the annotations, but `@NonNullByDefault` cannot be resolved. The import statement seem not to work. Any hints, @htreu? Sounded like simply adding it here should be fine.